src/xenia/base/cvar.cc

```diff
@@ -24,6 +27,20 @@ cxxopts::Options options("xenia", "Xbox 360 Emulator");
 std::map<std::string, ICommandVar*>* CmdVars;
 std::map<std::string, IConfigVar*>* ConfigVars;
 
+/*void ExitWithNotification(bool help) {
```

Comment:
There's commented code here, you should probably clean this up

Reply:
Sorry about that. Should be gone now.